go/worker/storage/committee/node.go

```diff
@@ -858,34 +875,97 @@ func (n *Node) consensusCheckpointSyncer() {
 	}
 	defer sub.Close()
 
+	var (
+		versions []uint64
+		blkCh    <-chan *consensus.Block
+		blkSub   pubsub.ClosableSubscription
+	)
 	for {
 		select {
```

Comment:
This leaks blkCh/blkSub if the first two cases are taken.

Reply:
Thanks, fixed!